sdk/go/examples_test.go

```diff
@@ -140,7 +139,10 @@ func ExampleContainer_WithMountedCache() {
 	}
 	defer client.Close()
 
-	cacheKey := "example-" + time.Now().Format(time.RFC3339)
+	// Do not add a timestamp to the cacheKey because it
```

Comment:
I guess we don't need this comment here since people reading this the first time won't know that a timestamp thingy was there in the first place :tongue: